testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueries.java

```diff
@@ -634,12 +634,8 @@ public void testShowSchemasLikeWithEscape()
     @Test
     public void testShowTables()
     {
-        Set<String> expectedTables = TpchTable.getTables().stream()
-                .map(TpchTable::getTableName)
-                .collect(toImmutableSet());
-
         MaterializedResult result = computeActual("SHOW TABLES");
-        assertThat(result.getOnlyColumnAsSet()).containsAll(expectedTables);
+        assertThat(result.getOnlyColumnAsSet()).containsAll(createdTpchTables());
```

Comment:
Don't expose this. Simpliy enumerat those tables, which are required by other test methods. this is sufficient from this test perspective